sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -728,11 +732,57 @@ export class ApplicationStudentsController extends BaseController {
         `Application id ${applicationId} not found or not in relevant status to get enrolment details.`,
       );
     }
+
```

Comment:
The update on API `getApplicationProgressDetails()` is missing. As per the AC it is required.

![image](https://github.com/bcgov/SIMS/assets/54600590/d04cdd12-1071-4f7b-9dde-258fea81ec27)